ONSdigital/response-operations-ui
# rops-design-improvements-ci-page-redesign

# What and why?

As part of the rOps redesign, the add and view CI EQ selection page needed to be updated. This required the following:

- Removal of the 'Link collection button' on the surveys page
- Move the link collection functionality to the add CI EQ section with additional error handling.
- Make alterations to the way selected CIs are shown when selected and unselected. Also including a tally
- Update the button to redirect to the CE overview page when CIs are successfully added or removed. Includes additional error handling
- Updated and moved tests

# How to test?
- Follow the instructions on the trello ticket to ensure that everything looks correct.
- Run unit tests
- Run acceptance tests.

# Trello
https://trello.com/c/ELDnXWO2/2344-rops-design-improvements-ci-page-redesign

# Jira
https://jira.ons.gov.uk/browse/RAS-266
